packages/functional/tests/ui/core/components/RadioGroup.spec.ts

```diff
@@ -12,21 +12,21 @@ test('Changing selection on radio button options should update query parameters'
 
   await expect(page).toHaveURL('shop-all/');
 
-  await page.getByRole('button', { name: 'Quick add' }).first().click();
+  await page.getByRole('link', { name: 'Quick add' }).first().click();
```

Comment:
@jorgemoya @chanceaclark Does this look correct (change from link -> button)?

<img width="336" alt="Screenshot 2024-02-13 at 2 56 19 PM" src="https://github.com/bigcommerce/catalyst/assets/58529077/a923185a-4c35-42b0-b6f8-9c466d93f736">
?

Reply:
Yes